### DISCUSSION TITLE
Is it possible to search in the document's description?

### DISCUSSION THREAD

**Discussion by teciON** [YetiForceCompany/YetiForceCRM#15805] [How to] [answered]
The search engine doesn't search in descriptions.

**Comment by mariuszkrzaczkowski** [comment]
Which search engine ??, specify the question and attach screenshots

**Comment by teciON** [comment]
Using the search engine (see the attachment, top red box) we want to search also in descriptions (marked in the screenshot at the bottom as "opis")

![crm](https://user-images.githubusercontent.com/99530823/159446713-2b416251-7673-4c36-91e7-4ae4ffc40794.jpg)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
This mechanism works only on selected fields set in the panel
![image](https://user-images.githubusercontent.com/10198654/159448630-c8d12d77-b818-47dd-8928-3cd619484528.png)

additionally, it has limitations / length limits
![image](https://user-images.githubusercontent.com/10198654/159448502-71d7ea3a-c7f5-452f-8c62-ff93bd676cf0.png)

**Reply by teciON** [reply]
Got it. Thank you.

**Reply by teciON** [reply]
.....and is it possible to double the number of characters searched?, because it is now set to ~ 236 characters.

**Reply by rskrzypczak** [reply]
Simple.. no.
